<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="group_control_panel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btn_advance_tick">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Advance Tick</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pause_autoadvance">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause/Auto Advance</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_inspect_world">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Inspect World</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_settings">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="inspector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>541</width>
      <height>291</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::TabPosition::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::TabShape::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_population">
     <attribute name="title">
      <string>Population</string>
     </attribute>
     <widget class="QListView" name="list_population">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_population_inspect">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Inspect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_settlements">
     <attribute name="title">
      <string>Settlements</string>
     </attribute>
     <widget class="QListView" name="list_settlements">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_settlements_inspect">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Inspect</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QOpenGLWidget" name="opengl_relations_map">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>701</width>
      <height>821</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_events_viewer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>541</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btn_showDetails">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>410</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Details</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>331</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_currentDate">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_refresh">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>370</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="dt_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Î”t:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_deltaTime">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_politics">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Politics</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_disasters">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Disasters</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_social">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Social</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_economy">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>120</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Economy</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_military">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>140</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Military</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_tech">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Technology</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cultural">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>180</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cultural</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_misc">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Misc</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1268</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Model_File"/>
    <addaction name="actionSave_Model"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Simulation"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Model_File">
   <property name="text">
    <string>Open Model File</string>
   </property>
  </action>
  <action name="actionSave_Model">
   <property name="text">
    <string>Save Model</string>
   </property>
  </action>
  <action name="actionClose_Simulation">
   <property name="text">
    <string>Close Simulation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
